auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Application.kt

```diff
@@ -23,6 +25,11 @@ fun main() {
             port = 8443
         }
         module {
+            install(CORS) {
+                allowHost("api.delta.test.communities.gov.uk")
```

Comment:
You forgot to use the env var here.

Something like 
```
                allowHost(Env.getOptionalOrDevFallback("API_ORIGIN", "http://localhost:8080")
```

Reply:
It's okay calling it API_ORIGIN because I guess that's what it is, but you could maybe put a code comment here just to say this is to allow access from Swagger.